source/docs/software/driverstation/driver-station-labview-dashboard.rst

```diff
@@ -0,0 +1,127 @@
+
+The Dashboard application installed and launched by the FRC Driver Station is a LabVIEW program designed to provide teams with basic feedback from their robot, with the ability to expand and customize the information to suit their needs. This Dashboard application uses Network Tables and contains a variety of tools that teams may find useful.
+
+LabVIEW Dashboard
+-----------------
+
+.. image:: images/driver-station-labview-dashboard/labview-dashboard.png
+
+The Dashboard is broken into two main sections. The left pane is for displaying a camera image. The right pane contains:
+
+- Drive tab that contains indicators for joystick and drive motor values (hooked up by default when used with LabVIEW robot code), a gyro indicator, an Autonomous selection text box, a connection indicator and some controls and indicators for the camera,
+- Basic tab that contains some default controls and indicators,
+- Custom tab for customizing the dashboard using LabVIEW
+- Test tab for use with Test Mode in the LabVIEW framework
+- Commands tab for use with the new LabVIEW C&C Framework
+- Checklist tab that can be used to create task lists to complete before and/or between matches
+- Variables tab that displays the raw Network Tables variables in a tree view format
+
+The LabVIEW Dashboard also includes Record/Playback functionality, located in the bottom right. More detail about this feature is included below under `Record/Playback`_.
+
+Camera Image and Controls
+-------------------------
+
+.. image:: images/driver-station-labview-dashboard/camera-image-and-controls.png
+
+The left pane is used to display a video feed from an Axis camera or USB camera located on the robot. For instructions on setting up the Axis Camera to work with this display :ref:`see here <docs/software/vision-processing/introduction/configuring-an-axis-camera:Configuring an Axis Camera>`. There are also some controls and indicators related to the camera in the right pane on the Drive tab and below the tab area:
+
+#. Camera Image Display
+#. Mode Selector - This drop-down allows you to select the type of camera display to use. The choices are Camera Off, USB Camera SW (software compression), USB Camera HW (hardware compression) and IP Camera (Axis camera). Note that the IP Camera setting will not work when your PC is connected to the roboRIO over USB.
+#. Camera Settings - This control allows you to change the resolution, framerate and compression of the image stream to the dashboard, click the control to pop-up the configuration.
+#. Bandwidth Indicator - Indicates approximate bandwidth usage of the image stream. The indicator will display green for "safe" bandwidth usage, yellow when teams should use caution and red if the stream bandwidth is beyond levels that will work on the competition field.
+#. Framerate - Indicates the approximate received framerate of the image stream.
+
+Drive
+-----
+
+.. image:: images/driver-station-labview-dashboard/drive.png
```

Comment:
Update picture for 2020

Reply:
Updated to: 
![image](https://user-images.githubusercontent.com/15172054/81518532-34df9e80-92f3-11ea-89d4-0f5c822cc265.png)